profiler/src/main/java/com/splunk/opentelemetry/profiler/JfrActivator.java

```diff
@@ -83,35 +81,42 @@ private boolean notClearForTakeoff(ConfigProperties config) {
       return true;
     }
 
-    Path outputDir = Paths.get(config.getString(CONFIG_KEY_PROFILER_DIRECTORY));
+    return false;
+  }
+
+  private boolean checkOutputDir(Path outputDir) {
     if (!Files.exists(outputDir)) {
       // Try creating the directory for the user...
       try {
         Files.createDirectories(outputDir);
       } catch (IOException e) {
-        return outdirWarn(outputDir, "does not exist and could not be created");
+        outdirWarn(outputDir, "does not exist and could not be created");
+        return false;
```

Comment:
These used to return true, so the semantic is changed. Does the caller account for this? What happens if the user specifies `keepfiles=true` but the output path is not available or writable or whatever? Do we still profile, but just not output files? Is this worth documenting maybe?

Reply:
Yes, the caller expects failure to return `false`. If user specifies `keepfiles=true` and output path is not writable then profiling will be enabled but files will not be written. This deviates from the current behaviour of turning profiling off when output path is not writable regardless of the `keepfiles` setting. `keepfiles` setting is not really useful for end users, I would not expect anybody using it, If users want jfr recordings they should use other tools to produce them.